pages/corona.tsx

```diff
@@ -20,13 +21,17 @@ export default function CoronaPage() {
       <Text>
         <p>Zu allererst: ruhig bleiben.</p>
         <p>
-          Wenn Du eine bestätigte Kontaktperson bist, wird sich das
-          Gesundheitsamt bei dir telefonisch innerhalb von 48 Stunden melden.
+          Wenn {isCareEnv ? 'Sie' : 'Du'} eine bestätigte Kontaktperson{' '}
+          {isCareEnv ? 'sind' : 'bist'}, wird sich das Gesundheitsamt bei{' '}
+          {isCareEnv ? 'Ihnen' : 'dir'} telefonisch innerhalb von 48 Stunden
+          melden.
         </p>
         <p>
-          Falls Du bereits typische Symptome wie Fieber, Müdigkeit und trockenen
-          Husten hast, solltest du vorsichtshalber in Haus-Quarantäne bleiben
-          und das Gesundheitsamt informieren.
+          Falls {isCareEnv ? 'Sie' : 'Du'} bereits typische Symptome wie Fieber,
```

Comment:
Ich glaub wenn das schon hier inline passiert, dann besser zwei komplette Strigs je nach `isCareEnv` statt so mitten in den Sätzen

Reply:
Ist halt ne schwierige balance, immer komplett getrennte bloecke ist auch echt viel